### ISSUE TITLE
New Crossbow - Blood Crossbow, Magic Assault Crossbow, Archers Crossbow

### ISSUE THREAD

**Issue by davidjreese** [IronWillGames/Dransik#958]
Traecneh new crossbow an Enbar's crossbows

- [x] Create animation
- [x] Create type
- [x] Setup stats
- [x] Add to shop inventory (buy list)
- [ ] Add to monster drops

<img width="314" alt="Screenshot 2024-02-15 at 6 27 08 PM" src="https://github.com/IronWillGames/Dransik/assets/669018/4763cfb6-1117-4b11-b5f8-1d8789789f22">

**Comment by PatrickKaHoTang** [comment]
Change Magic Assault Crossbow into the following to fill the gaps:

- [x] +1 Crossbow
- [x] +2 Crossbow
- [x] +3 Crossbow
- [x] +4 Crossbow
- [x] +5 Crossbow

**Comment by PatrickKaHoTang** [comment]
Stats for new crossbows set up - will need new types ready to import

**Comment by davidjreese** [comment]
these were added and new export provided

**Comment by davidjreese** [comment]
@PatrickKaHoTang - I added all these bows to the generic treasure tables and to the shops.

Most of the 1-3 bows were in 1 table due to how the tables are clustered with different drop rates based on the bow level.

Let me know if you want specific things on specific monsters (or if you want any removed froim the generic tables) and which go where if so.

Closing this out today if I dont hear back
